### PR TITLE
Restore the ability to set only votes via json

### PR DESCRIPTION
**Pull request by phate89**
The bump of json version is needed right?

### CODE REVIEW

File: xbmc/interfaces/json-rpc/VideoLibrary.cpp [hunk 1]
```diff
@@ -1126,8 +1126,11 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
...
   {
     details.SetRating(parameterObject["rating"].asFloat());
-    if (ParameterNotNull(parameterObject, "votes"))
-      details.SetVotes(StringUtils::ReturnDigits(parameterObject["votes"].asString()));
+    updatedDetails.insert("ratings");
+  }
+  if (ParameterNotNull(parameterObject, "votes"))
+  {
+    details.SetVotes(StringUtils::ReturnDigits(parameterObject["votes"].asString()));
     updatedDetails.insert("ratings");
```
Comment (Razzeee) [inline xbmc/interfaces/json-rpc/VideoLibrary.cpp]: Can you add a small comment here why this is needed? Might be confusing and not everybody might want to look what's happening here.

Rest seems fine.

**Comment by anaconda** [comment]
Thanks @phate89.
I don't expect any client to handle this specific 3-days change, but it makes sense to bump API version again.

Btw I'm still seeing something odd after #8080. I'm using Milhouse's texturecache script (https://github.com/MilhouseVH/texturecache.py/blob/master/texturecache.py) to update ratings and votes every day.
I've investigated this further toni...this morning and
`texturecache.py imdb movies | texturecache.py set`
updates movies correctly with this fix, but
`texturecache.py imdb tvshows | texturecache.py set`
doesn't update TV shows and episodes. Looks like something like https://github.com/xbmc/xbmc/pull/8080/files#diff-dbf978b6cac348a2af93d3beec420e43R2272 is missing, so the database is never updated.
Like I said, grab that script, check what needs updating with
`texturecache.py imdb tvshows`
and try to update the database with
`texturecache.py imdb tvshows | texturecache.py set`
(API will report `"result": "OK"`), then check again with
`texturecache.py imdb tvshows`.
The script can produce a log file when passed `@logfile=filename` (`texturecache.py ... @logfile=mylog`).

**Comment by Razzeee** [comment]
Thank you. I'll address the comment I asked for myself.

**Comment by phate89** [comment]
Thanks @Razzeee

**Pull request merged**